csharp/src/SeedLang/X/SeedPythonDentLexer.cs

```diff
@@ -145,27 +148,32 @@ internal class SeedPythonDentLexer : SeedPythonLexer {
       EmitToken(eofToken);
     }
 
-    private bool TryPopFirstToken(out IToken first) {
-      if (_tokens.Count == 0) {
-        first = null;
-        return false;
+    private void ScanVTag(IToken comment) {
+      if (comment.Text.Length > 1) {
+        Debug.Assert(comment.Text[0] == '#');
+        var inputStream = new AntlrInputStream(comment.Text.Substring(1));
+        var lexer = new SeedPythonLexer(inputStream);
+        var tokens = lexer.GetAllTokens();
+        if (tokens.Count > 0 && tokens[0].Type == SeedPythonParser.VTAG_START) {
+          for (int i = 0; i < tokens.Count; i++) {
+            // Adds back '#' for the VTAG_START token.
```

Comment:
Please double-check if the following adding back logic is meeting the expectation:

Here is a common case:

```
    public void TestVTag() {
      string source = "# \t\t\t\t[[ Print ]]\nprint(1)";
      var expectedTokens = new string[] {
          $"[@-1,0:7='# [[',<{SeedPythonParser.VTAG_START}>,1:0]",
          ... ...
      };
```

I added 4 "\t" to the above test case between "# " and "[[". The first output token has its range set to 0:7 but its actual length is 4.

Reply:
Great catch. Fixed this bug.